File: program2_2.java
// Accept one number from user and if the number is less than 10 print Hello otherwise print Demo

import java.util.*;

class DisplayX
{
    public void Display(int iNo)
    {
        if((iNo<10))
        {
            System.out.println("Hello");
        }
        else
        {
            System.out.println("Demo");
        }
    }
}
class program2_2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter Number:");
        int iValue= sobj.nextInt();

        DisplayX dobj = new DisplayX();
        dobj.Display(iValue);
    }
}

File: program2.2.c
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//  Problem Statement: Accept number from user and print that number of star(*) using while loop and updator
//  Input: 5
//  Output: * * * * *
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>


void Display(int iNo)
{
    if(iNo<0)
    {
        iNo=-iNo;
    }

    int iCnt=1;

    while(iCnt<=iNo)
    {
        printf("*\n");
        iCnt++;
    }
}
int main()
{
    int iValue=0;

    printf("Enter Number:\n");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}